CaseManagement/models.py

```diff
@@ -47,11 +54,11 @@ def get_default_pk(cls):
             defaults=dict(dateCreated="1111-12-16 22:12", dateLastModified="1111-12-16 22:12", createdBy="None", lastModifiedBy="None", caseIdentifier="None"),
         )
         return tag.pk
-
-    # TODO: Make a dictionary containing all the properties on this model listed above
+        
+    # TODO: ASK CHUCK ABOUT THIS ONE. Make a dictionary containing all the properties on this model listed above
         # and return the json.dumps() of that. e.g. { "name": self.name, ... }
     def __str__(self) -> str:
-        # TODO: This isn't a great way to do it. Example of identifier `OBSID-0001` with this:
+        # TODO: AND THIS ONE This isn't a great way to do it. Example of identifier `OBSID-0001` with this:
```

Comment:
This isn't related to your current task; leaving the todos is good but this was actually why the Project with GitHub issues was created because we're gonna have a bunch of todos and keeping track in code which is yours, which is mine, and which is for later would be sketchy. Just easier to track as GitHub issues